termsaver

```diff
@@ -180,75 +180,109 @@ if __name__ == '__main__':
             # Handles issues with file or directory reading errors
             #
             error_number = errno.ENOENT
-            print _('Oops! Could not find path for %(path)s. %(msg)s') % \
-                {'path': e.path, 'msg': e.message}
+            error_message = ""
+            if hasattr(e, 'message'):
+                error_message = e.message
+            else:
+                error_message = e
+            print ('Oops! Could not find path for %(path)s. %(msg)s') % \
+                {'path': e.path, 'msg': error_message}
 
         elif isinstance(e, exception.InvalidOptionException):
             #
             # Handles issues with invalid options
             #
             error_number = errno.EINVAL
+            error_message = ""
+            if hasattr(e, 'message'):
+                error_message = e.message
+            else:
+                error_message = e
             suf = ''
+
             if e.option_name not in (None, ''):
                 suf = " '%s'" % e.option_name
-            print _('Oops! Invalid use of option%(suf)s. %(msg)s.%(help)s') % \
+            print ('Oops! Invalid use of option%(suf)s. %(msg)s.%(help)s' % \
                 {'suf': suf,
-                 'msg': e.message,
-                 'help': _(' See --help for details.')}
```

Comment:
This changes the context of the 'help' value... the _() function returns a string in the language of the computer (if available), and your changes changes it to a tuple.

Reply:
Honestly then this may be an unnecessary change. I may have been a little overzealous

Reply:
Reverted edit in new update.